lib/compile_time.flow

```diff
@@ -0,0 +1,34 @@
+export {
+	// Normally all these functions are eliminated from program before passing it to backends.
+
+	// Speculative function, used to ask to compute a constant value during compilation.
+	// In case a compile-time computation fails, the error will be reported
+	compileTime(v: ?) -> ? {
+		v
+	}
+
+	// Speculative function, used to ask to compute a constant value during compilation.
+	// In case a compile-time computation fails, the default value is returned
+	compileTimeDef(v: ?, def: ?) -> ? {
+		def
+	}
+
+	// Speculative function, used to make output of computed constant values during compilation.
```

Comment:
Can't you just use println in the compile time part? I do not understand the need for this.